pyaedt/emit_core/results/revision.py

```diff
@@ -306,9 +306,8 @@ def get_active_frequencies(self, radio_name, band_name, tx_rx_mode, units=""):
             Name of the radio/emitter.
         band_name : str
            Name of the band.
-        tx_rx : :class:`emit_constants.TxRxMode`
-            Specifies whether to get ``tx`` or ``rx`` radio freqs. The default
-            is ``None``, in which case both ``tx`` and ``rx`` freqs are returned.
+        tx_rx_mode : :class:`emit_constants.TxRxMode`
+            Specifies whether to get ``tx`` or ``rx`` radio freqs.
         units : str, optional
             Units for the frequencies. The default is ``None`` which uses the units
```

Comment:
```suggestion
            Units for the frequencies. The default is ``None``, in which case the units
```

Reply:
@PipKat - We should be able to put this tweak in a future PR. (@jsalant22 @apollo-crum)